model_analyzer/reports/report_manager.py

```diff
@@ -450,11 +450,12 @@ def _build_summary_table(self,
                 row = [
                     model_config.get_field('name'), max_batch_size,
                     model_config.dynamic_batching_string(), instance_group_str,
-                    measurement.get_metric_value('perf_latency_p99'),
-                    measurement.get_metric_value('perf_throughput'),
-                    measurement.get_metric_value('cpu_used_ram'),
-                    measurement.get_metric_value('gpu_used_memory'),
-                    round(measurement.get_metric_value('gpu_utilization'), 1)
+                    measurement.get_non_gpu_metric_value('perf_latency_p99'),
+                    measurement.get_non_gpu_metric_value('perf_throughput'),
+                    measurement.get_non_gpu_metric_value('cpu_used_ram'),
+                    measurement.get_gpu_metric_value('gpu_used_memory'),
```

Comment:
Some of these are lists, and some are scalar, right? Are we just fixing single model here and pushing the multi-model change to later story?

Reply:
Correct.